Recursion/C/permutation.c

```diff
@@ -0,0 +1,26 @@
+  
+// Code in C programming language to generate the permutations for the set of numbers entered .
+
+#include <stdio.h>
+/*function to find factorial*/
+int fact(int n){
+    int factorial=n;
+    for(int i=n-1;i>1;i--){
+        factorial*=i;
+    }
+    return factorial;
+    }
+  /*function to find permutation*/  
+    int npr(int n,int r){
+        return fact(n)/fact(n-r);
```

Comment:
In your code its just showing the no of permutations . 
Also provide the list of them . 
Your code should be such as if I enter 3 digits it should display the ways in which it could be arranged .